proto/ligato/vpp/wireguard/wireguard.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package ligato.vpp.wireguard;
+
+option go_package = "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/wireguard;vpp_wg";
+
+message Peer {
+  // Public-key base64
+  string public_key = 1;
+
+  // Peer UDP port
+  uint32 port = 2;
+
+  // Keepalive interval (sec)
+  uint32 persistent_keepalive = 3;
+
+  // VRF ID
+  // The IP table in which 'endpoint' is reachable
+  uint32 vrf_id = 4;
```

Comment:
Isn't VRF given by the tunnel interface?
You can obtain VRF from the interface metadata: https://github.com/ligato/vpp-agent/blob/master/plugins/vpp/ifplugin/ifaceidx/ifaceidx.go#L63

Reply:
yep it should